simcore/simulation/library/anchor.cpp

```diff
@@ -195,6 +197,20 @@ void Anchor::UpdateAnchorPositionToBond() {
   }
   UpdatePeriodic();
 }
+/*Creates Vector that has different binding rates for parallel and anti-parallel bonds*/
+std::vector<double> Anchor::CreateOrientationArray(int n_neighbors){
```

Comment:
This function should be void that takes a vector by reference. See comment below. Also, the name should be changed for clarity on what it's doing. e.g. void UpdatePolarAffinityBinding(std::vector\<double\> &binding_rates)